Python 3.8.7 (tags/v3.8.7:6503f05, Dec 21 2020, 17:59:51) [MSC v.1928 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
====================== RESTART: D:/python files/list44.py ======================
enter a number67 77 75 4 46
Traceback (most recent call last):
  File "D:/python files/list44.py", line 1, in <module>
    num =int(input("enter a number"))
ValueError: invalid literal for int() with base 10: '67 77 75 4 46'
>>> 
>>> 
====================== RESTART: D:/python files/list44.py ======================
enter a number2
Traceback (most recent call last):
  File "D:/python files/list44.py", line 2, in <module>
    print(list(num))
TypeError: 'int' object is not iterable
>>> 
>>> 
====================== RESTART: D:/python files/list44.py ======================
enter a number2, 3, 4, 5
Traceback (most recent call last):
  File "D:/python files/list44.py", line 1, in <module>
    num =int(input("enter a number"))
ValueError: invalid literal for int() with base 10: '2, 3, 4, 5'
>>> 
====================== RESTART: D:/python files/list44.py ======================
enter a number22 33 44 55 
Traceback (most recent call last):
  File "D:/python files/list44.py", line 1, in <module>
    num =int(input("enter a number"))
ValueError: invalid literal for int() with base 10: '22 33 44 55 '
>>> 
====================== RESTART: D:/python files/list44.py ======================
enter a number65 56 43 76
['65 56 43 76']
>>> 
====================== RESTART: D:/python files/list44.py ======================
Traceback (most recent call last):
  File "D:/python files/list44.py", line 2, in <module>
    print(index[1])
NameError: name 'index' is not defined
>>> 
====================== RESTART: D:/python files/list44.py ======================
>>> 2
2
>>> 
====================== RESTART: D:/python files/list44.py ======================
Traceback (most recent call last):
  File "D:/python files/list44.py", line 3, in <module>
    print(index[0])
NameError: name 'index' is not defined
>>> 
====================== RESTART: D:/python files/list44.py ======================
10
20
30
40
Traceback (most recent call last):
  File "D:/python files/list44.py", line 5, in <module>
    for i in range(len(item)):
NameError: name 'item' is not defined
>>> 
==================== RESTART: D:/python files/equalcheck.py ====================
Traceback (most recent call last):
  File "D:/python files/equalcheck.py", line 5, in <module>
    print(false)
NameError: name 'false' is not defined
>>> 
==================== RESTART: D:/python files/equalcheck.py ====================
False
True
>>> 
==================== RESTART: D:/python files/equalcheck.py ====================
yes
>>> 
==================== RESTART: D:/python files/equalcheck.py ====================
>>> 